internal/terraform/module_test.go

```diff
@@ -782,6 +784,49 @@ func TestLoadComments(t *testing.T) {
 	}
 }
 
+func TestReadComments(t *testing.T) {
+	tests := []struct {
+		name         string
+		path         string
+		fileName     string
+		readComments bool
+		expected     string
+	}{
+		{
+			name:         "Validate description when 'ReadComments' is false",
+			path:         "read-comments",
+			fileName:     "variables.tf",
+			readComments: false,
+			expected:     "",
+		},
+		{
+			name:         "Validate description when 'ReadComments' is true",
+			path:         "read-comments",
+			fileName:     "variables.tf",
+			readComments: true,
+			expected:     "B description",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			assert := assert.New(t)
+			options := NewOptions()
+			options.ReadComments = tt.readComments
```

Comment:
Just one more question @khos2ow. for me, it took a while to figure out that a `true` boolean flag cannot be overriden (to  `false`) with `options.With` and `.WithOverwrite`. [This comment]((https://github.com/terraform-docs/terraform-docs/blob/master/internal/terraform/module_test.go#L43) helped. do you think it'll be helpful to add it here as well ?  I did not want to add an extra comment if it's not needed.

Reply:
That's a very good observation, but you can leave it out for now as `Options` and `Settings` will be deprecated in favor of `Config` as part of #510.